superliminal.asl

```diff
@@ -214,6 +231,13 @@ init
                 dllSize != 1744384)   // steam v1.10.2021.4.18 & v1.10.2021.5.10
                 version = "2021mp";
         }
+        else if (game.ProcessName == "SuperliminalGog")
+        {
+            string dllPath = modules.First().FileName + "\\..\\SuperliminalGOG_Data\\Managed\\Assembly-CSharp.dll";
+            long dllSize = new System.IO.FileInfo(dllPath).Length;
+            if (dllSize == 1793024) // gog v1.10.2021.11.12.858.39
```

Comment:
This will not be future-proof, since new versions will have different DLL sizes. It would be better to check that the size is **not** equal to one of the pre-multiplayer versions, but that requires you to have access to files from earlier versions.

Reply:
By previous versions do you mean 2019, 2020, 2021 or builds?

Reply:
I mean every game versions starting from v1.10.2020.12.10 and **before** the multiplayer update (v1.10.2021.11.5). I listed the ones from Steam and EGS [here](https://github.com/loriswit/asl/blob/main/superliminal.md#2021) (with their respective DLL sizes).

If you look at the part of the script for Steam and EGS, it checks whether `dllSize` is **different** from any of theses versions. If it is indeed different, then it assumes we're in the multiplayer update (v1.10.2021.11.5 and later). That way, the script is future proof (we don't know the size of the DLL of future versions yet).

Reply:
Updated size check per your suggestion, but i wasn't able to get v1.10.2021.4.1x